crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -97,7 +97,7 @@ def finished?
 
     def save
       progress_file_path.open("w") do |f|
-        f.write(JSON.pretty_generate(progress))
+        f.write(YAML.dump(progress))
```

Comment:
Shoudln't we rather go back to symbolizing the string values, like we've had before in https://github.com/crowbar/crowbar-core/commit/b7909220d9f5572b4f695d1b81f093143e72cb8f (and reverted by https://github.com/crowbar/crowbar-core/commit/52589eee5b121fa198570967cb2bcfe830feb0e8)